Jenkinsfile

```diff
@@ -72,10 +72,8 @@ def buildManifest = {String manifest, String bitbake_image ->
     sh "vagrant destroy -f || true"
 }
 
-node("DockerCI") {
-    buildManifest("pelux-intel.xml", "core-image-pelux")
-}
+def builders = [:]
+builders['core'] = { node("DockerCI") { buildManifest("pelux-intel.xml", "core-image-pelux") }}
+builders['qt']   = { node("DockerCI") { buildManifest("pelux-intel-qt.xml", "core-image-pelux-qt") }}
```

Comment:
It might also be nice to add a comment describing what the parallel command does. Unfortunately, I do not think groovy syntax speaks for it self.